dfm_tools/download.py

```diff
@@ -54,6 +61,9 @@ def download_ERA5(varkey,
     period_range = pd.period_range(date_min,date_max,freq='M')
     print(f'retrieving data from {period_range[0]} to {period_range[-1]} (freq={period_range.freq})')
     
+    #make sure the data fully covers the desired spatial extent. Download 1 additional grid cell (resolution is 1/4 degrees) in both direction
+    longitude_min, longitude_max, latitude_min, latitude_max = extend_domain(lon_min=longitude_min, lon_max=longitude_max, lat_min=latitude_min, lat_max=latitude_max, delta_latlon=1/4)
```

Comment:
I suggest to remove the `extend_domain()` function since and replace these lines with:
```
#make sure the data fully covers the desired spatial extent. Download 1 additional grid cell (resolution is 1/4 degrees) in both directions
lon_min -= 1/4
lon_max += 1/4
lat_min -= 1/4
lat_max += 1/4
```
I know it is against re-using code in functions, but this one is so simple and we hardcode delta_latlon anyway.